remarkable_sync.py

```
@@ -0,0 +1,364 @@
+    except Exception as e:
+        print(f"❌ Failed to load config: {e}")
+        return
+    
+    try:
+        # Initialize sync
+        sync = RemarkableSync(config)
+        
+        # Perform sync
+        uploaded, downloaded = sync.full_sync()
+        
+        print(f"\\n✅ Sync complete!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)